src/rt/backtrace/dwarf.d

```diff
@@ -12,6 +12,11 @@
 
 module rt.backtrace.dwarf;
 
+private import rt.backtrace.execinfo;
+
+static if (hasExecinfo)
+{
```

Comment:
Could be `static if (hasExecinfo):`, then you don't need the closing parenthesis at the end of the file (and make it clear that the module is otherwise empty).